lib/stat_tracker.rb

```diff
@@ -113,4 +113,47 @@ def worst_coach(season)
 
     coach_results.min_by { |k, v| v }[0]
   end
+
+  def calculate_team_accuracies(season)
+    team_accuracies = {}
+
+    game_teams_in_season = @game_teams_data.select do |game_team|
+      game_id = game_team[:game_id]
+      game = @games_data.find { |game| game[:game_id] == game_id }
+
+      game && game[:season] == season
+    end
+
+    game_teams_in_season.each do |game_team|
```

Comment:
instead of this implementation; maybe make team accuracies to be a `{ team_id: [goals, shots] }`; 
you just sum all goals and all shots and then calculate ratio just once at the end as opposed to making array of ratios and then averaging them.

Reply:
going to stick with what we have for now